eval.go

```diff
@@ -202,9 +187,14 @@ func (e *CallExpr) eval(app *App, args []string) {
 			}
```

Comment:
On current master, in https://github.com/gokcehan/lf/blob/master/eval.go#L208, we're creating a file, but do not `return` on failure. And subsequent code will try to write to it.